src/api/transaction/suspay-finish.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+'use strict';
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const Transaction = utils.common.core.Transaction;
+
+function createSusPayFinishTransaction(account, payment) {
+  validate.address(account);
+  validate.suspendedPaymentFinish(payment);
+
+  const transaction = new Transaction();
+  transaction.susPayFinish({
+    from: account,
+    owner: payment.owner,
+    offerSequence: payment.offerSequence
```

Comment:
Seems like swinging from frivolous to overly stingy. At 38/256 I think we're some ways from being forced to use **Offer**Sequence for Payments. We haven't painted ourselves that far into the corner yet.

Also, have these actually been used historically: 
```
SF_U32 const sfStampEscrow         = make::one<SF_U32::type>(&sfStampEscrow,         STI_UINT32, 22, "StampEscrow");
SF_U32 const sfBondAmount          = make::one<SF_U32::type>(&sfBondAmount,          STI_UINT32, 23, "BondAmount");
SF_U32 const sfOperationLimit      = make::one<SF_U32::type>(&sfOperationLimit,      STI_UINT32, 29, "OperationLimit");
```

?

At one point @scottschurr and I asked the data team if they could search for TicketID uses. Maybe we could recycle an ordinal.